pkgs/applications/misc/bdt/default.nix

```
@@ -0,0 +1,21 @@
+{ pkgs ? import <nixpkgs> { }, lib ? pkgs.lib }:
```

```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
}:
```

`callPackage` will fill these arguments automatically.